test/test_linalg.py

```diff
@@ -5534,9 +5569,18 @@ def test_addmm(self, device, dtype):
                   *[torch.bfloat16] if TEST_WITH_ROCM or SM53OrLater else []))
     @dtypes(*floating_types_and(torch.bfloat16))
     @tf32_on_and_off(0.05)
-    def test_addmm_activation(self, device, dtype):
+    def test_addmm_relu(self, device, dtype):
         self._test_addmm_impl(torch._addmm_activation, "relu", device, dtype)
 
+    @precisionOverride({torch.double: 1e-8, torch.float: 1e-4, torch.bfloat16: 0.6,
+                        torch.half: 1e-1, torch.cfloat: 1e-4, torch.cdouble: 1e-8})
```

Comment:
i'm surprised the torch.half bounds are so high here, but this is a prior issue, no need to fix on your pr

Reply:
My understanding is that these tests are not run on `torch.half` (only on `float32`, `float64`, and `bfloat16` on CUDA). So maybe it's a copy / paste artifact.